common-service-core/src/main/java/com/ca/mfaas/config/ApiInfo.java

```diff
@@ -49,37 +49,37 @@
     //TODO: move to EurekaMetadataParser
     public Map<String, String> generateMetadata(String serviceId) {
         Map<String, String> metadata = new HashMap<>();
-        String encodedGatewayUrl = getEncodedGatewayUrl(gatewayUrl);
+        String encodedGatewayUrl = UrlUtils.getEncodedUrl(gatewayUrl);
```

Comment:
It seems that methods were moved to UrlUtils but they also remain still here.

Reply:
I don't see them in ApiInfo class. 
On the other hand, there is still TODO to move method generateMetadata 
 to EurekaMetadataParser class.

Reply:
Moved method generateMetadata from ApiInfo to EurekaMetadataParser